src/scss/components/_navbar.scss

```diff
@@ -1,3 +1,6 @@
+@use 'uswds-core' as *;
+@use 'usa-nav';
```

Comment:
oooo namespaced imports in SCSS... would we ever want to update some of these to be `uswds-core.xyz` in the future?

Reply:
We could. I think that's one of the main appeals of the namespaces, though I'm still finding it difficult to get used to it. I don't like the verboseness of prefixing everything, though I suppose the "everything is global" of older Sass has its own flaws.